README.md

```diff
@@ -1,65 +1,69 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+> A basic "Hello Microverse" page that illustrate the use of tools and best practices for future Microverse projects.
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Linters
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
+[Live Demo Link - coming soon]()
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
+- Node and NPM
+- Web browser (Google chrome)
+- Code Editor (VSCode)
+
+### Development (Running locally)
 
-### Install
+- Clone the project
 
-### Usage
+```bash
+git clone https://github.com/Hamzaoutdoors/hello-microverse-cr.git
 
-### Run tests
+```
 
-### Deployment
+To run StyleLint by itself, you may run the lint task:
 
+```bash
+npx stylelint "**/*.{css,scss}"
+```
 
+Or to automatically fix issues found (where possible):
 
-## Authors
+```bash
+npx stylelint "**/*.{css,scss}" --fix
+```
 
-👤 **Author1**
+You can also check against Webhint:
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+```bash
+npx hint --telemetry=off .
+```
 
-👤 **Author2**
+## 👤 Author
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- Github: [@Hamzaoutdoors](https://github.com/Hamzaoutdoors)
+- Twitter: [@ellaouzihamza]()
```

Comment:
Great work adding your contact links to the [README ](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)file, but your Twitter link is broken. Would you kindly fix it? Having a well-written readme file without any broken links helps users to know more about you and to navigate and identify essential elements of your project quickly.